docs/sources/mimir/configure/configure-blocked-queries.md

```diff
@@ -30,6 +30,40 @@ To set up runtime overrides, refer to [runtime configuration]({{< relref "./abou
 The order of patterns is preserved, so the first matching pattern will be used.
 {{% /admonition %}}
 
+## Formatting queries to block
```

Comment:
We should avoid gerunds (verbs ending with "ing")

I'm not sure "Format(ting) queries to block" really makes sense.

from the context of the page, because we are already talking about blocking queries, I think the following suggestion makes sense:

```suggestion
## Format queries
```